### DISCUSSION TITLE
Use meters or millimeters in module and station definitions?

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#118] [Q&A] [answered]
Why have these parameters unit of measure in meter, while the length of the module is in millimetres? 

It makes more sense to define this in millimetres. 

![image](https://user-images.githubusercontent.com/57874863/131733088-4ccb44dc-1389-451c-bca6-13e4c03e8b1d.png)

**Comment by fjallemark** [comment, marked as answer]
I think units of measure in meters are easier to read and enter, because it often has fewer digits than a measure in mm. The reason modules are measured in mm was a request from FREMO. However, I may revert that and use meters everywhere.

**Reply by AlexEhn** [reply]
I suggest to stay with mm.
